### ISSUE TITLE
Obsolete nucleic acid-templated transcription

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#24383]
nucleic acid-templated transcription is an unnecessary grouping class, it only has 2 children, 'DNA-templated transcription' and 
'RNA-templated transcription', these would be fine direct children of 'RNA bioynthetic process' and keep 'part of' some 'gene expression' from the parent. 

Thanks, Pascale

**Comment by deustp01** [comment]
Out of ignorance, are there any non-templated forms of RNA synthesis that are biologically important? If there were, then a grouping class to distinguish them from the templated ones might be useful.

**Comment by ukemi** [comment]
Poly-A tails?

**Comment by pgaudet** [comment]
We are not proposing to obsolete 'RNA biosynthetic process'  (GO:0032774) - just the its child, the grouping term  'nucleic acid-templated transcription'

**Comment by deustp01** [comment]
Right. The question is whether we gain something by keeping children of 'RNA biosynthetic process' (GO:0032774) as grouping terms to distinguish, e.g., templated from non-templated RNA biosynthesis.

**Comment by paolaroncaglia** [comment]
@pgaudet 
Hi, just for the records, this is the thread that lead to the creation of GO:0097659 nucleic acid-templated transcription:
https://github.com/geneontology/go-ontology/issues/11369
See you soon,
Paola
